.github/workflows/check-contributor-pack.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   check_contributor_pack:
     runs-on: ubuntu-latest
-    if: github.repository == 'demisto/content' && startsWith(github.head_ref, 'contrib/') == false && startsWith(github.head_ref, 'to-merge/') == false && contains(github.head_ref, 'xsoar-bot-contrib-ContributionTestPack') == false
+    if: github.repository == 'demisto/content' && startsWith(github.base_ref, 'contrib/') == false && startsWith(github.head_ref, 'to-merge/') == false && contains(github.head_ref, 'xsoar-bot-contrib-ContributionTestPack') == false
```

Comment:
Are we sure this disables this flown on the external PRs and only runs on the internal ones?

Reply:
No

Reply:
changed to better if clause..